packages/ts/react-form/test/useForm.spec.tsx

```diff
@@ -373,131 +374,84 @@ describe('@vaadin/hilla-react-form', () => {
     });
   });
 
-  function TeamFormPlayer({ model }: { model: PlayerModel }) {
-    const { field, value, invalid, ownErrors } = useFormPart(model);
+  describe('arrays', () => {
+    let players: Signal<Player[]>;
+
+    function TeamFormPlayer({ model }: { model: PlayerModel }) {
+      const { field, value, invalid, ownErrors } = useFormPart(model);
+
+      return (
+        <div>
+          <input data-testid={`lastName.${value!.id}`} type="text" {...field(model.lastName)} />
+          <output data-testid={`validation.lastName.${value!.id}`}>
+            {invalid ? ownErrors.map((e) => e.message).join(', ') : 'OK'}
+          </output>
+        </div>
+      );
+    }
 
-    return (
-      <div>
-        <input data-testid={`lastName.${value!.id}`} type="text" {...field(model.lastName)} />
-        <output data-testid={`validation.lastName.${value!.id}`}>
-          {invalid ? ownErrors.map((e) => e.message).join(', ') : 'OK'}
-        </output>
-      </div>
-    );
-  }
+    function TeamForm({ initialPlayers }: { initialPlayers: Player[] }) {
+      const { field, model, read } = useForm(TeamModel);
+      const name = useFormPart(model.name);
+      const { items, value, setValue } = useFormArrayPart(model.players);
 
-  function TeamForm({
-    testAction,
-  }: {
-    testAction({
-      read,
-      value,
-      setValue,
-    }: {
-      read(value: Team | null | undefined): void;
-      value: Player[] | undefined;
-      setValue(value: Player[] | undefined): void;
-    }): void;
-  }) {
-    const { field, model, read } = useForm(TeamModel);
-    const name = useFormPart(model.name);
-    const { value, setValue } = useFormPart(model.players);
-
-    useEffect(() => {
-      testAction({ read, value, setValue });
-    }, []);
+      players = useSignal(initialPlayers);
```

Comment:
Do we really need a signal here? 🤔

Reply:
No, it was just a way to simplify code, but I'm open to alternatives. It allows me to react to further changes to the player array.